src/containers/bookmark/bookmark.scss

```diff
@@ -0,0 +1,13 @@
+.bookmark-container {
+  cursor: pointer;
+  position: absolute;
+  z-index: 100;
+
+  &-add-icon {
+    font-size: 50px;
+  }
+  &-remove-icon {
+    font-size: 50px;
+    color: #84a2ff;
```

Comment:
You can create a variable `$color: #84a2ff` at the top and reuse it here.

Reply:
Done